pkg/kms/webkms/remotekms.go

```diff
@@ -36,15 +36,15 @@ var logger = log.New("aries-framework/kms/webkms")
 
 type createKeystoreReq struct {
 	Controller         string `json:"controller,omitempty"`
-	OperationalVaultID string `json:"pperationalvaultid,omitempty"`
+	OperationalVaultID string `json:"operationalVaultID,omitempty"`
```

Comment:
i am confused where the term operational is coming from for a generic remote kms.

Reply:
operational keys are the user's "working" keys in opposed to the "service" keys used for supporting tasks (e.g., delegate/recipient keys)

I agree that this is too implementation-specific, so it makes sense to rename it to a more generic term, like "edvVaultID" or just "vaultID".